lib/localeapp/rails/mimic_rails_missing_translation_display.rb

```diff
@@ -12,7 +12,9 @@ def self.included(o)
         alias :old_rails_call :call
         def call(exception, locale, key, options)
           locale, key = old_rails_call(exception, locale, key, options).split(', ')
-          keys = I18n.normalize_keys(locale, key, options[:scope])
+          keys = I18n.normalize_keys(locale, key, options[:scope]).map do |key|
+            ERB::Util.html_escape(key.to_s)
```

Comment:
Doing this here unfortunately doesn't cover all cases (because this class is not always loaded, as you can see in [lib/localeapp/rails.rb](https://github.com/Locale/localeapp/blob/master/lib/localeapp/rails.rb#L20-L24)).
It would be better to do it in [lib/localeapp/exception_handler.rb](https://github.com/Locale/localeapp/blob/master/lib/localeapp/exception_handler.rb#L13) instead by replacing `[locale, key].join(', ')` by `[locale, key].map{ |key| ERB::Util.html_escape(key.to_s) }.join(', ')` on lline 13.